src/orange/detail/UnitInserter.cc

```diff
@@ -373,6 +382,38 @@ VolumeRecord UnitInserter::insert_volume(SurfacesRecord const& surf_record,
     return output;
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Process a single oriented bounding zone record.
+ */
+void UnitInserter::process_obz_record(VolumeRecord* vol_record,
+                                      OrientedBoundingZoneInput const& obz_input)
+{
+    CELER_EXPECT(obz_input);
+
+    OrientedBoundingZoneRecord obz_record;
+
+    // Set half widths
+    auto inner_hw_id = fast_real3s_.push_back(
+        calc_half_widths(calc_bumped_(obz_input.inner)));
+    auto outer_hw_id = fast_real3s_.push_back(
+        calc_half_widths(calc_bumped_(obz_input.outer)));
+    obz_record.hw_ids = {inner_hw_id, outer_hw_id};
+
+    // Set offsets
+    auto inner_offset_id = insert_transform_(VariantTransform{
+        std::in_place_type<Translation>, calc_center(obz_input.inner)});
+    auto outer_offset_id = insert_transform_(VariantTransform{
+        std::in_place_type<Translation>, calc_center(obz_input.outer)});
+    obz_record.offset_ids = {inner_offset_id, outer_offset_id};
+
+    // Set transformation
+    obz_record.transform_id = obz_input.transform_id;
```

Comment:
I don't understand why there's an extra set of variant transforms, especially on top of an *additional* transform. The offsets are only ever translations (or zero).